### ISSUE TITLE
New "di" command to download video/playlist info

### ISSUE THREAD

**Issue by Vrihub** [mps-youtube/mps-youtube#746]
## Issue / Suggestion

New feature: a `di` command to download the video/playlist information screens to a .txt file (in `ddir`).

See PR #745 and its initial message for details about the implementation.

It works for me, anyway, please review it and suggest improvements!

**Comment by ids1024** [comment]
What would be the use case of this? Having some basic metadata to go along with downloaded videos?

**Comment by Razesdark** [comment]
I feel like this kind of specific functionality would be best suited as a plugin. Maybe we should take another look at refactoring the app to support plugins?

**Comment by Vrihub** [comment]
> What would be the use case of this? Having some basic metadata to go along with downloaded videos?

Yes, in my case, I often need to save the data contained in the video description and having a dedicated command is more convenient than manual copy/paste. Maybe someone else is interested too, so I shared the code. :)

(BTW initially I thought about adding a config option to _always_ save the related .txt file whenever a da/dv command is executed, but the implementation seemed to require a lot more work)
